GA/HeaderFiles/ConstraintDipole.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+bool ConstraintDipole(float R, float L)
+{
+    // Set variables
+    bool intersect = true;
+
+    // distances in cm
+    float max_radius = 10.0;
+    float min_radius = 0.25;
+    float max_length = 10000.0;
+    float min_length = 200.0;
+
+    // Check to see if antenna meets requirements
+    if (R > max_radius || R < min_radius  || L > max_length || L < min_length)
+    {
+        intersect = true;
+    }
+    else
+    {
+        intersect = false;
+    }
+
+    // Return interesect condition
+    return intersect;
+}
```

Comment:
Missing newline at the end of the file.  You can configure your editor to add it automatically when you save the file.